pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -2047,7 +2047,7 @@ protected CompletableFuture<Void> internalSetOffloadThresholdInSecondsAsync(long
 
         validateNamespacePolicyOperationAsync(namespaceName, PolicyName.OFFLOAD, PolicyOperation.WRITE)
                 .thenApply(v -> validatePoliciesReadOnlyAccessAsync())
```

Comment:
@poorbarcode Thanks for finding the issue and fixing this.

Isn't there a similar bug with the previous line of `validatePoliciesReadOnlyAccessAsync`?